Objective-C/CBLDatabase.h

```diff
@@ -493,25 +494,14 @@ __deprecated_msg("Use [[database defaultCollection] setDocumentExpirationWithID:
 __deprecated_msg("Use [[database defaultCollection] getDocumentExpirationWithID:] instead.");
 
 
-#pragma mark - Query
-
-/**
- Creates a Query object from the given query string.
- 
- @param query Query expression
- @param error error On return, the given query string is invalid.
- @return query created using the given expression string.
- */
-- (nullable CBLQuery*) createQuery: (NSString*)query error: (NSError**)error;
-
 #pragma mark -- Scopes
 
 /**
  Get scope names that have at least one collection.
  
  @note: The default scope is exceptional as it will always be listed even though there are
  no collections under it. */
-- (NSArray*) scopes;
+- (NSArray<CBLScope*>*) scopes;
```

Comment:
nullable and could return error now.

Reply:
Please check the latest email about API change.

Here at the list of API that could return an error.
```
**Collection (Section 5.2, 5.5)**
Indexes() -> [String]
getDocument(id: String) -> Document?

**Scope (Section 5.4)**
getCollections() -> [Collection]
getCollection(name: String) -> Collection?
 
**Database (Section 5.6.2 , 5.6.3)**
getDefaultCollection() -> Collection?
getDefaultScope() -> Scope
getScopes() -> [Scope]
getScope(name: String) -> Scope?
getCollections(scope: String? = kDefaultScopeName) -> [Collection]
getCollection(name: String, scope: String? = kDefaultScopeName) -> Collection?
```